README.md

```diff
@@ -1,70 +1,51 @@
-# Getting Started with Create React App
+# BookStore CMS
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet)
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+- React (CRA)
 
-### `npm start`
+## sneak peek
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+![Application screenshot]() // Coming soon
```

Comment:
- [OPTIONAL] The link to your screenshot is broken, please kindly fix the issue as this will make your readme file look more professional.